src/app/blog/(index)/page.jsx

```diff
@@ -0,0 +1,47 @@
+import AppearanceEngineering from 'components/pages/blog/appearance-engineering';
+import FeaturedPostsList from 'components/pages/blog/featured-posts-list';
+import PostsList from 'components/pages/blog/posts-list';
+import ReleaseNotesList from 'components/pages/blog/release-notes-list';
+import VideoList from 'components/pages/blog/video-list';
+import SubscribeForm from 'components/pages/blog-post/subscribe-form';
+import { getWpBlogPage } from 'utils/api-posts';
+
+async function getReleaseNotesData() {
+  const res = await fetch(`${process.env.NEXT_PUBLIC_DEFAULT_SITE_URL}/api/release-notes`);
+
+  if (!res.ok) {
+    throw new Error('Failed to fetch data');
+  }
+
+  return res.json();
+}
+
+export default async function BlogPage() {
+  const releaseNotes = await getReleaseNotesData();
```

Comment:
If release notes will not load, the build will be failed. Is it ok?

Reply:
I think this is normal because we store these files statically and we always have access to them, an error can only occur if these files no longer exist or if one of the files used the wrong markup, if the error does occur, we need to check the data, because not only this page depends on them.